.agents/skills/e2e-container/scripts/start-envtest.sh

```
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Start the envtest Kubernetes cluster and wait for the kubeconfig.
+
+set -euo pipefail
+
+export KUBEBUILDER_ASSETS=$(setup-envtest use -p path)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)